Dockerfile

```diff
@@ -0,0 +1,89 @@
+FROM ubuntu:18.04
+RUN apt-get update
```

Comment:
`apt-get update && upgrade` should be in 1 RUN in docker, for caching purposes. otherwise the update may have no effect.

Reply:
Okay. I'll make the change.